lms/djangoapps/learner_dashboard/tests/mixins.py

```diff
@@ -0,0 +1,177 @@
+'''
+The Mixins used for unit tests for learner dashboard
+'''
+import httpretty
+import json
+
+from openedx.core.djangoapps.programs.models import ProgramsApiConfig
+
+
+class ProgramsDataMixin(object):
```

Comment:
Why not use the mixin defined in https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/programs/tests/mixins.py#L33? I would prefer that we could avoid duplicating these.